boards/arm/stm32f072_eval/doc/index.rst

```diff
@@ -112,7 +110,7 @@ Pin Mapping
 STM32F072-EVAL Discovery kit has 6 GPIO controllers. These controllers are responsible for pin muxing,
 input/output, pull-up, etc.
 
-For mode details please refer to `STM32F072-EVAL board User Manual`_.
+For mode details please refer to STM32F072-EVAL board User Manual.
```

Comment:
ditto here, if board is obsolete we should probably remove it @erwango

Reply:
sure, but that is beyond the scope of this PR.

Reply:
yep, just mentioning

Reply:
> ditto here, if board is obsolete we should probably remove it @erwango

I propose to review this once [board tiers policy](https://github.com/zephyrproject-rtos/zephyr/issues/38566) is finalized.